src/components/charts/line-chart.tsx

```diff
@@ -178,6 +181,20 @@ export class LineChart extends BaseComponent<ILineProps, ILineState> {
       },
       annotation: {
         annotations: chartData.formattedAnnotations
+      },
+      tooltips: {
+        enabled: true,
+        callbacks: {
+          label(tooltipItems: any, data: any) {
+            return `(${tooltipItems.xLabel}, ${Math.round(tooltipItems.yLabel * 100) / 100})`;
+          },
```

Comment:
I chose to do the rounding here to preserve the data as long as we can and only adjust the final displayed text in the tooltip popup.